packages/dialtone-tokens/build-layered.js

```diff
@@ -0,0 +1,400 @@
+      const darkVars = extractVars(darkContent);
+
+      const combined = `/**
+ * Do not edit directly, this file was auto-generated.
+ * High contrast overrides for both light and dark modes
+ */
+
+/* High contrast - Light mode */
+[data-dt-mode="light"][data-dt-contrast="high"] {
+${lightVars}
+}
+
+/* High contrast - Dark mode */
+[data-dt-mode="dark"][data-dt-contrast="high"] {
+${darkVars}
+}`;
+
+      await fs.writeFile(`${contrastDir}/tokens-high-contrast.css`, combined);
+      await fs.unlink(`${contrastDir}/${highLightFile}`);
+      await fs.unlink(`${contrastDir}/${highDarkFile}`);
+
+      console.log('Generated high contrast override file');
+    }
+  } catch {
+    console.log('No high contrast files found, skipping...');
+  }
+}
+
+/**
+ * Output build summary
+ *
+ * Shows a readable summary of what got built.
+ *
+ * Includes:
+ * - Output location (dist/css/layered/)
+ * - Core token files with sizes
+ * - Number of theme override files
+ * - High contrast location
+ *
+ * Gives visibility into build output without exploring files manually.
+ */
+function outputBuildSummary(stats) {
+  console.log('\n✅ Optimized build complete!');
+  console.log('\nFinal structure:');
+  console.log('  dist/css/layered/');
+  console.log('    ├── tokens-core.css (50KB) [typography, spacing, components]');
+  console.log('    ├── tokens-base-colors.css (177KB)');
+  console.log('    ├── tokens-dp-colors.css (628KB) [base theme]');
+  console.log(`    ├── themes/ (${stats.themeCount} override files)`);
+  console.log('    └── contrast/');
+  console.log('        └── tokens-high-contrast.css [high contrast overrides]');
+}
+
+/**
+ * Main function to generate optimized theme files
+ *
+ * Runs the complete layered token build. Multi-step workflow to optimize
+ * CSS token file sizes while keeping full functionality.
+ *
+ * Steps:
+ * 1. generateFullTokens()
+ *    Generate full theme files (~640KB each) with Style Dictionary. Temporary.
+ *
+ * 2. generateThemeOverrides()
+ *    Compare each theme to DP base, extract ONLY differences.
+ *    Write tiny override files (~0.5KB) to themes/ directory.
+ *    Track stats (usually 95%+ reduction).
+ *
+ * 3. cleanupFullThemeFiles()
+ *    Delete temporary full files.
+ *
+ * 4. processHighContrast()
+ *    Combine high-light and high-dark into one file.
+ *
+ * 5. outputBuildSummary()
+ *    Show what got built and where to find it.
+ *
+ * Why not output diffs directly?
+ * Style Dictionary can't easily output only differences. Simpler to generate
+ * complete files, calculate diffs, then discard temps.
+ */
+async function main() {
+  const outputDir = 'dist/css/layered';
+
+  await generateFullTokens(outputDir);
+  const stats = await generateThemeOverrides(outputDir);
+  await cleanupFullThemeFiles(outputDir);
+  await processHighContrast(outputDir);
+  outputBuildSummary(stats);
+}
+
+// Export for use in build.js
+export { main as buildLayeredTokens };
```

Comment:
this is redundant, call the function `buildLayeredTokens`

Reply:
Done [ccda6e4](https://github.com/dialpad/dialtone/pull/945/commits/ccda6e43f10735f56096069e514cbee8e2b278f4)